kategory-core/src/main/kotlin/kategory/data/ListKW.kt

```diff
@@ -1,7 +1,7 @@
 package kategory
 
 @higherkind
-@deriving(Monad::class, Traverse::class, MonoidK::class)
+@deriving(Monad::class, Traverse::class, MonoidK::class, MonadCombine::class)
```

Comment:
To be consistent with the rest of typeclasses in and derived instances we should here list all of the instances individually... Functor, Applicative, Monad, SemigroupK, MonoidK,...